docs/platforms/python/tracing/span-metrics/examples.mdx

```diff
@@ -0,0 +1,593 @@
+---
+title: Example Instrumentation
```

Comment:
The content on this page is quite advanced, I would emphasize this in the title

```suggestion
title: Advanced Examples
```

I also somewhat question the wisdom of including examples like these in the docs. It might be better to maintain these examples in a separate GitHub repository somewhere. I am unsure whether they are even needed in the first place, regardless of whether they live in the docs or in a repo, it could be a burden to maintain these

Reply:
We heard quite a bit of feedback from the field and customers about wanting examples in the docs of what custom span instrumentation would look like. I feel pretty strongly that we should have some examples in our docs - especially when expect people to instrument custom spans fairly regularly. 

Can you talk me through a bit more on the perspective of why not include practical examples here?